UI/MauiEmbedding/MauiCommunityToolkitApp/MauiCommunityToolkitApp/MauiCommunityToolkitApp.csproj

```diff
@@ -1,90 +1,79 @@
-    <PackageReference Include="Xamarin.AndroidX.Collection.Ktx" VersionOverride="$(AndroidXCollectionVersion)" />    
-  </ItemGroup>
+        error NETSDK1148: A referenced assembly was compiled using a newer version of Microsoft.Windows.SDK.NET.dll.
+        Please update to a newer .NET SDK in order to reference this assembly.
 
-	
-	<Choose>
-		<When Condition="$(IsWinAppSdk)">
-			<PropertyGroup>
-				
-				<!--
-				If you encounter this error message:
+      This means that the two packages below must be aligned with the "build" version number of
+      the "Microsoft.Windows.SDK.BuildTools" package above, and the "revision" version number
+      must be the highest found in https://www.nuget.org/packages/Microsoft.Windows.SDK.NET.Ref.
+    -->
+    <!-- <WindowsSdkPackageVersion>10.0.22621.28</WindowsSdkPackageVersion> -->
 
-					error NETSDK1148: A referenced assembly was compiled using a newer version of Microsoft.Windows.SDK.NET.dll.
-					Please update to a newer .NET SDK in order to reference this assembly.
+    <!--
+      UnoFeatures let's you quickly add and manage implicit package references based on the features you want to use.
+      https://aka.platform.uno/singleproject-features
+    -->
+    <UnoFeatures>
+      Hosting;
+      Toolkit;
+      Logging;
+      MauiEmbedding;
+      MVUX;
+      Configuration;
+      Http;
+      Serialization;
+      Localization;
+      Navigation;
+    </UnoFeatures>
+  </PropertyGroup>
+  <!--<ItemGroup>
+    <None Remove="Presentation\DrawingViewContent.xaml" />
+  </ItemGroup>-->
 
-				This means that the two packages below must be aligned with the "build" version number of
-				the "Microsoft.Windows.SDK.BuildTools" package above, and the "revision" version number
-				must be the highest found in https://www.nuget.org/packages/Microsoft.Windows.SDK.NET.Ref.
-				-->
-				<!-- <WindowsSdkPackageVersion>10.0.22621.28</WindowsSdkPackageVersion> -->
-			</PropertyGroup>
-		</When>
-		<Otherwise>
-			<ItemGroup>
-				<PackageReference Include="Uno.WinUI.Lottie" />
+  <ItemGroup>
+    <ProjectReference Include="..\MauiCommunityToolkitApp.MauiControls\MauiCommunityToolkitApp.MauiControls.csproj" />
+  </ItemGroup>
 
-				<!-- Include all images by default - matches the __WindowsAppSdkDefaultImageIncludes property in the WindowsAppSDK -->
-				<Content Include="Assets\**;**/*.png;**/*.bmp;**/*.jpg;**/*.dds;**/*.tif;**/*.tga;**/*.gif" Exclude="bin\**;obj\**;**\*.svg" />
-				<Page Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
-				<Compile Update="**\*.xaml.cs">
-					<DependentUpon>%(Filename)</DependentUpon>
-				</Compile>
-				<PRIResource Include="**\*.resw" />
-			</ItemGroup>
-		</Otherwise>
-	</Choose>
+  <ItemGroup>
+    <PackageReference Include="CommunityToolkit.Maui" />
+    <PackageReference Include="CommunityToolkit.Mvvm" />
+    <PackageReference Include="Microsoft.Maui.Controls" />
+    <PackageReference Include="Microsoft.Maui.Controls.Compatibility" />
```

Comment:
You need to add `<PackageReference Include="Microsoft.Maui.Graphics" />`

With the MauiVersion as the package version in the Directory.Packages.props